museum_of_dreams_project/settings/base.py

```diff
@@ -155,6 +156,7 @@
             ["TextColor", "BGColor"],
             ["Smiley", "SpecialChar"],
             ["Popout"],
+            ["Html5video"],
```

Comment:
I think we could move this and replace "Flash" next to "Image" with this (we don't even have Flash enabled)